src/arch/amdgpu/vega/insts/instructions.hh

```diff
@@ -43915,78 +43916,232 @@ namespace VegaISA
 
         void execute(GPUDynInstPtr) override;
     }; // Inst_VOP3P__V_PK_MOV_B32
+       //
+    class Inst_VOP3P_MAI__V_MFMA_I32_16X16X16I8 : public Inst_VOP3P_MAI {
+    public:
+      Inst_VOP3P_MAI__V_MFMA_I32_16X16X16I8(InFmt_VOP3P_MAI *);
+      ~Inst_VOP3P_MAI__V_MFMA_I32_16X16X16I8();
+
+      int
+      getNumOperands() override
+      {
+          return numDstRegOperands() + numSrcRegOperands();
+      } // getNumOperands
+
+      int numDstRegOperands() override { return 1; }
+      int numSrcRegOperands() override { return 3; }
+
+      int
+      getOperandSize(int opIdx) override
+      {
+          switch (opIdx) {
+            case 0: // src0 "A"
```

Comment:
might be useful to have slightly more detailed comments about what A, B, and C mean